docs/source/reference/hf.rst

```diff
@@ -0,0 +1,34 @@
+HuggingFace Models in EVA
+======================
+
+This section provides an overview of how you can use out-of-the-box HuggingFace models in EVA.
+
+
+Creating UDF from HuggingFace
+-----
+EVA supports UDFS similar to `Pipelines <https://huggingface.co/docs/transformers/main_classes/pipelines>`_  in HuggingFace. 
+
+.. code-block:: sql
+
+    CREATE UDF IF NOT EXISTS HFObjectDetector
+    TYPE  HuggingFace
+    'task' 'object-detection'
+    'model' 'facebook / detr-resnet-50'
```

Comment:
Is this another way to create UDF? Internally it will be translated to how EVA normally creates UDF.

Reply:
Yeah that's the goal of the design, users can use HF UDF's without writing code and EVA treats it similar to any other UDF.